homeassistant/components/switch/ihc.py

```diff
@@ -0,0 +1,76 @@
+"""IHC switch platform.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.ihc/
+"""
+from xml.etree.ElementTree import Element
+
+import voluptuous as vol
+
+from homeassistant.components.ihc import (
+    validate_name, IHC_DATA, IHC_CONTROLLER)
+from homeassistant.components.ihc.ihcdevice import IHCDevice
+from homeassistant.components.switch import (SwitchDevice, PLATFORM_SCHEMA)
```

Comment:
Imported names that fit a single line don't need parenthesis.

Reply:
Done